### PR TITLE
Update init.lua

### PR DESCRIPTION
**Pull request by anniop**
***************************************************************************
**NOTE**
Please verify that the `base repository` above has the intended destination!
Github by default opens Pull Requests against the parent of a forked repository.
If this is your personal fork and you didn't intend to open a PR for contribution
to the original project then adjust the `base repository` accordingly.
**************************************************************************

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -207,15 +207,16 @@ vim.api.nvim_create_autocmd('TextYankPost', {
 -- [[ Install `lazy.nvim` plugin manager ]]
 --    See `:help lazy.nvim.txt` or https://github.com/folke/lazy.nvim for more info
 local lazypath = vim.fn.stdpath 'data' .. '/lazy/lazy.nvim'
-if not vim.uv.fs_stat(lazypath) then
+if not vim.loop.fs_stat(lazypath) then
```
Comment (iton0) [inline init.lua:210]: using loop instead of uv results in a warning that fs_stat is an undefined field

**Review by iton0 — CHANGES_REQUESTED** [review]

**Comment by iton0** [comment]
@anniop was this meant for your personal repo? If it was please close this pr.

File: init.lua [hunk 2]
```diff
@@ -207,15 +207,16 @@ vim.api.nvim_create_autocmd('TextYankPost', {
...
   local lazyrepo = 'https://github.com/folke/lazy.nvim.git'
   local out = vim.fn.system { 'git', 'clone', '--filter=blob:none', '--branch=stable', lazyrepo, lazypath }
   if vim.v.shell_error ~= 0 then
     error('Error cloning lazy.nvim:\n' .. out)
   end
-end ---@diagnostic disable-next-line: undefined-field
+end
 vim.opt.rtp:prepend(lazypath)
 
+
```
Comment (iton0) [inline init.lua:219]: delete this empty line

**Comment by anniop** [comment]
I was jusy forking the repo for my own customisations but mistakenly opened a pr so sorry for that

**Pull request closed without merging**